src/solution.c

```diff
@@ -47,6 +47,9 @@ systime_t last_dgnss;
 double soln_freq = 10.0;
 u32 obs_output_divisor = 2;
 
+bool_t output_spp_when_rtk_avail = false;
```

Comment:
Doesn't look like this is used yet? If not then should probably remove it until the functionality is implemented